docs/readthedocs/models/input/participant/evcs.rst

```diff
@@ -1,5 +1,123 @@
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| Cee32ASocket                  | cee32                                         | 22            | AC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| Cee63ASocket                  | cee63                                         | 43            | AC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| ChargingStationType1          | cst1, stationtype1, cstype1                   | 7.2           | AC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| ChargingStationType2          | cst2, stationtype2, cstype2                   | 43            | AC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| ChargingStationCcsComboType1  | csccs1, csccscombo1                           | 11            | DC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| ChargingStationCcsComboType2  | csccs2, csccscombo2                           | 50            | DC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| TeslaSuperChargerV1           | tesla1, teslav1, supercharger1, supercharger  | 135           | DC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| TeslaSuperChargerV2           | tesla2, teslav2, supercharger2                | 150           | DC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| TeslaSuperChargerV3           | tesla3, teslav3, supercharger3                | 250           | DC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+
+
+Custom Types
+""""""""""""
+While the provided standard types should be suitable for most scenarios, providing an individual type for a specific
+scenario might be necessary. To do so, a custom type can be provided instead of a common id. This custom type is tested
+against the following regex :code:`(\w+\d*)\s*\(\s*(\d+\.?\d+)\s*\|\s*(AC|DC)\s*\)`, or more generally, the custom
+type string has to be in the following syntax::
+
+    <Name>(<Apparent Power in kVA>|<AC|DC>) e.g. FastCharger(50|DC) or SlowCharger(2.5|AC)
+
+Please note, that in accordance with :code:`edu.ie3.datamodel.models.StandardUnits` the apparent power is expected to
```

Comment:
I would suggest to have an different version of the regex:
`(\w+\d*)\ ?\(\ ?(\d+(?:\.\d+)?\ ?[kMG]?W)\ ?\|\ ?(AC|DC)\ ?\)`

- `\s+` would also allow tabs and stuff, additionally in any amount. `\ ?` would only allow one blank at most
- Explicitly demand for a power string (e.g. '50 kW', '5000 W', '0.5GW', that can be parsed to quantity directly.

Reply:
While I really like this idea and its verbosity I would prefer not implementing this approach for now as this requires several things to be changed incl. 

- ensuring that there is always a white space between the value and the unit (`50 kW` can be parsed by the Unit API, `50kW` throws an exception)
- requires casting of the resulting value using `asType(Power.class)`
- still requires conversion to `StandardUnits.S_RATED` using `to(StandardUnits.S_RATED)`
- several test adaptions

All these changes for a small verbosity improvement I would prefer keeping the existing solution and if there will be a point where having a more explicit mapping is required we could address this.